src/common/syncjournaldb.cpp

```diff
@@ -953,20 +958,26 @@ bool SyncJournalDb::deleteFileRecord(const QString &filename, bool recursively)
         // if (!recursively) {
         // always delete the actual file.
 
-        if (!_deleteFileRecordPhash.initOrReset(QByteArrayLiteral("DELETE FROM metadata WHERE phash=?1"), _db))
-            return false;
+        {
+            const auto query = _queryManager.get(PreparedSqlQueryManager::DeleteFileRecordPhash, QByteArrayLiteral("DELETE FROM metadata WHERE phash=?1"), _db);
+            if (!query) {
+                return false;
+            }
 
-        qlonglong phash = getPHash(filename.toUtf8());
-        _deleteFileRecordPhash.bindValue(1, phash);
+            qlonglong phash = getPHash(filename.toUtf8());
```

Comment:
the method returns qint64, which is probably the same as qlonglong, but should probably be consistent at some point.